README.md

```diff
@@ -7,39 +7,51 @@
 
 ## Exercise
 
-Take a look at each of the code samples below. Each example will show you some
-code, as well as either the return value of what is logged to the console.
+Each of the following code samples accesses an API with CityBike data from
+data.gov. Take a look at each sample and answer the associated question(s).
 
-1. How does this piece of code work?
-2. Why does the code do what it does?
-3. Given this code sample, what can you learn or describe about JavaScript
+To run the code samples yourself, open the browser DevTools and copy/paste the
+code into the console.
 
 ### Example 1
 
-```javascript
-const url = "https://data.cityofnewyork.us/api/views/p94q-8hxh" // CityBike Data from data.gov
-const data = fetch(url)
+Why does the following code snippet result in the error shown below? How can we
+fix it?
 
-console.log(data) // Promise {<pending>}
+```javascript
+const url = "https://data.cityofnewyork.us/api/views/p94q-8hxh"
+fetch(url).then(console.log)
+/*
+Refused to connect to 'https://data.cityofnewyork.us/api/views/p94q-8hxh' because it violates the document's Content Security Policy.
+Promise {<rejected>: TypeError: Failed to fetch
+    at <anonymous>:2:1}
+Uncaught (in promise) TypeError: Failed to fetch
+    at <anonymous>:2:1
+*/
```

Comment:
I think its just supposed to print a `Promise`, under ideal circumstances. I do notice that students get CORS errors sometimes. 

Under ideal circumstances, its supposed to print out a Promise object, which provides the talking point for what Promises are

<img width="1467" alt="Screen Shot 2022-09-16 at 9 36 40 AM" src="https://user-images.githubusercontent.com/1832658/190652288-43c62a65-5f06-4a82-aafd-c812b9c821e3.png">

Reply:
Thanks @ckesurf ! My intention here was to have them (hopefully) recognize that it's a CORS error and that to clear it they need to navigate to the API in the browser and run the request from there. If you don't think that's a reasonable expectation, or if it's not a helpful discussion for them to have, I can just add instructions for clearing the error instead. Thoughts?